src/pages/MyMoviesPage.tsx

```diff
@@ -1,21 +1,50 @@
 import React, { useState, useEffect } from "react";
+import { Container, Header } from "semantic-ui-react";
 
 import LayoutContainer from "../components/LayoutContainer";
+import MovieAdder from "../components/MovieAdder";
+import MovieList from "../components/MovieList";
 import { Path } from "../constants";
-import { AuthenticatedPageProps, User } from "../types";
+import { fetchMovies } from "../network/requests";
+import { AuthenticatedPageProps, MoviePreference } from "../types";
 
 const MyMoviesPage = ({
   sessionUser,
   setSessionUser
 }: AuthenticatedPageProps): JSX.Element => {
+  const [movies, setMovies] = useState<Array<MoviePreference>>([]);
+  const fetchUserMovies = () => {
+    fetchMovies(sessionUser, setMovies);
+  };
+  useEffect(fetchUserMovies, []);
```

Comment:
This should have dependencies in its array because we should re-fetch if the user changed or if the setMovies changed.

```suggestion
  useEffect(fetchUserMovies, [sessionUser, setMovies]);
```